Makefile

```diff
@@ -188,6 +188,14 @@ githooks-commit:
 githooks-push:
 	@if [ ! -e .git/hooks/pre-push ]; then ln -s ../../bin/pre-push .git/hooks/pre-push; fi
 
+# generate a new shrinkwrap
+shrinkwrap:
+	@rm -rf node_modules
```

Comment:
could we fail this target entirely if the runner isn't on the correct node and npm? should we?